src/styles/card.scss

```diff
@@ -0,0 +1,77 @@
+.catalog__card {
+  margin: 0 $card-horizontal-margin 50px;
+}
+
+.card {
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
```

Comment:
this should be assigned to all elements with * selector

Reply:
Okay. Fixed